game/plugin/skills/woodcutting/src/axe.kt

```diff
@@ -0,0 +1,23 @@
+package org.apollo.game.plugin.skills.woodcutting
+
+import org.apollo.game.model.Animation;
+
+//Animation IDs: https://www.rune-server.ee/runescape-development/rs2-client/configuration/272373-emote-gfx-id-list.html
+enum class Axe(val id: Int, val level: Int, val animation: Animation, val pulses: Int) {
+    RUNE(1359, 41, Animation(867), 3),
+    ADAMANT(1357, 31, Animation(869), 4),
+    MITHRIL(1355, 21, Animation(871), 5),
+    BLACK(1361, 11, Animation(873), 6),
+    STEEL(1353, 6, Animation(875), 6),
+    IRON(1349, 1, Animation(877), 7),
+    BRONZE(1351, 1, Animation(879), 8)
+}
+
+val AXES = enumValues<Axe>()
```

Comment:
Where's `enumValues` coming from?

Reply:
My code.  I was seeing some interesting issues iterating over Axe.values().  Might have been local only.

Reply:
No, I was having the same issue. So I saw your approach and that seemed to help. I was getting a static initialization error.

Reply:
So, if I move the vals that the enum uses to above the enum, then I get this:
```
SEVERE: Exception occurred during pulse!
java.lang.ExceptionInInitializerError
	at org.apollo.game.plugin.skills.woodcutting.WoodKt.<clinit>(wood.kt:22)
	at Woodcutting_plugin$3.invoke(woodcutting.plugin.kts:112)
	at Woodcutting_plugin$3.invoke(woodcutting.plugin.kts:16)
	at org.apollo.game.plugin.kotlin.KotlinPlayerHandlerProxyTrait$DefaultImpls.handleProxy(KotlinPluginScript.kt:81)
	at org.apollo.game.plugin.kotlin.KotlinMessageHandler.handleProxy(KotlinPluginScript.kt:97)
	at org.apollo.game.plugin.kotlin.KotlinMessageHandler.handleProxy(KotlinPluginScript.kt:97)
	at org.apollo.game.plugin.kotlin.KotlinMessageHandler.handle(KotlinPluginScript.kt:103)
	at org.apollo.game.message.handler.MessageHandlerChain.notify(MessageHandlerChain.java:57)
	at org.apollo.game.message.handler.MessageHandlerChainSet.notify(MessageHandlerChainSet.java:45)
	at org.apollo.game.session.GameSession.handlePendingMessages(GameSession.java:96)
	at org.apollo.game.service.GameService.pulse(GameService.java:162)
	at org.apollo.game.GamePulseHandler.run(GamePulseHandler.java:37)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.IllegalArgumentException: Parameter specified as non-null is null: method org.apollo.game.plugin.skills.woodcutting.Wood.<init>, parameter objects
	at org.apollo.game.plugin.skills.woodcutting.Wood.<init>(wood.kt)
	at org.apollo.game.plugin.skills.woodcutting.Wood.<clinit>(wood.kt:11)
	... 19 more


```